packages/claude-hooks/README.md

```
@@ -351,25 +461,63 @@ pnpm run test
 1. **Auto-Config Loading** - Hooks automatically load `.claude/hooks/{hookName}.config.json`
-2. **Environment Variables** - Override any JSON setting using `CLAUDE_HOOKS_*` variables
-3. **CLI Arguments** - Override both JSON and environment settings
-4. **Defaults** - Built-in fallbacks if nothing else is specified
+2. **Environment Variable Substitution** - Use `${VAR_NAME}` syntax in JSON config files
+3. **Environment Variables** - Override any JSON setting using `CLAUDE_HOOKS_*` variables
+4. **CLI Arguments** - Override both JSON and environment settings
+5. **Defaults** - Built-in fallbacks if nothing else is specified
 
 ### Configuration Priority
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)